client/src/Components/Avatar/_tests_/Avatar.test.tsx

```diff
@@ -50,11 +49,13 @@ describe('Avatar', () => {
     expect(avatar).toHaveAttribute('src', defaultAvatarImage);
   });
 
-  it('Should set image source to user avatar after logging in', () => {
-    store.dispatch(login([exampleUser]));
-    renderAvatarImage();
+  //add later when login will be ready
 
-    const avatar = screen.getByRole('img');
-    expect(avatar).toHaveAttribute('src', testImg);
-  });
+  // it('Should set image source to user avatar after logging in', () => {
+  //   store.dispatch(login([exampleUser]));
+  //   renderAvatarImage();
+
+  //   const avatar = screen.getByRole('img');
+  //   expect(avatar).toHaveAttribute('src', testImg);
+  // });
```

Comment:
Tego chyba nie potrzebujemy

Reply:
potrzebbjemy, bo bedzie trzeba zrobic logowanie i wtedy bedzie trzeba od nowa pisac testy dlatego zakomentowalam i dodalam komentarz //add later when login will be ready

Reply:
To w takim razie git